account_loan/models/account_loan.py

```diff
@@ -29,22 +29,19 @@ def _default_company(self):
         required=True,
         readonly=True,
```

Comment:
This readonly is necessary?

Example of the migration guide:
https://github.com/odoo/odoo/pull/104741/files#diff-1e3bd6be3bfb83a37ec9fb800ce8b1c95afe0be90ff792874ae7299c320a2f6eL162-L163